manifests/cf-manifest/cloud-config/010-cf-networks.yml

```diff
@@ -1,7 +1,6 @@
 ---
-
 networks:
-- name: cf1
+- name: cf
   subnets:
     - range: 10.0.16.0/24
       reserved:
```

Comment:
We, don't want to reserve 10.0.16.1 ? And the dynamic range goes to 10.0.16.255 ( which is broadcast ) not .254